### PR TITLE
Use github-post-release for changelog generation

### PR DESCRIPTION
**Pull request by jlegrone**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -82,5 +83,9 @@
     "after": [
       "ember-computed-decorators"
     ]
+  },
+  "release": {
+    "analyzeCommits": "simple-commit-message",
```
Comment (alexdiliberto) [inline package.json]: @jlegrone Do we not need to `yarn add simple-commit-message`?

**Comment by jbailey4** [comment]
ping @jlegrone

**Comment by crodriguez1a** [comment]
@jlegrone Should we keep this open?

**Comment by jlegrone** [comment]
Yeah just closing this for now.

**Pull request closed without merging**